src/handlers/init.js

```diff
@@ -0,0 +1,46 @@
+const { Cache } = require('memory-cache')
+const { Tracer } = require('@mojaloop/event-sdk')
+
+const { logger } = require('../lib/logger')
+const Config = require('../lib/config')
+const Database = require('../data/cachedDatabase')
+const modelFactory = require('../model')
+const createConsumers = require('./createConsumers')
+const QuotingHandler = require('./QuotingHandler')
+
+let db
+let consumersMap
+
+const startFn = async (handlerList) => {
+  const config = new Config()
+
+  db = new Database(config)
+  await db.connect()
+  const isDbOk = await db.isConnected()
+  if (!isDbOk) throw new Error('DB is not connected')
+
+  const { quotesModelFactory, bulkQuotesModelFactory } = modelFactory(db)
+
+  const handler = new QuotingHandler({
+    quotesModelFactory,
+    bulkQuotesModelFactory,
+    config,
+    logger,
+    cache: new Cache(),
+    tracer: Tracer
+  })
+
+  consumersMap = await createConsumers(handler.handleMessages, handlerList)
```

Comment:
There is file setup.js which handles the registration of handlers in central ledger. It also abstracts some other functionalities like running migration scripts, initializing cache ...etc. Can we follow the same approach here instead of calling starting consumer here directly?

Reply:
First of all, there is no step for running migration during handlers start:
https://github.com/mojaloop/central-ledger/blob/main/src/handlers/index.js#L162

Next, you can do/run/start whatever you need in _startFn_:
![image](https://github.com/mojaloop/quoting-service/assets/16463615/4f23348b-80f1-4eca-a3f4-c6c1c96e91e0)

And the last, there is no setup.js in quoting-service, the start process of API was initially implemented without it:
https://github.com/mojaloop/quoting-service/blob/main/src/server.js#L201

So i didn't change anything, and I tried to follow the same approach for starting handlers (using start fn). So could you, please, tell me what's wrong with this approach, and why do we need to change existing in this service approach?

Reply:
If its not there, lets add it to align it with central-ledger so that we can add if there are any initialisations later. Just like you added startFn, I am proposing to use the same naming convention and structure across different repos. If you think it takes time, then that's ok, we can standardise it later to align with central-ledger.

Reply:
I would suggest, first of all, to check GP tests for now, and make them pass. Coz it's the most important thing, as for me. After that we can have a separate call (with all devs), and discuss which exactly parts of our services we need to standardize